radiomanager/radiomanager/__init__.py

```diff
@@ -34,14 +23,47 @@ class TESTING_CONFIG:
     SQLALCHEMY_DATABASE_URI = "sqlite://"
 
 
-def create_app(config_obj=PRODUCTION_CONFIG):
+def create_app(config_obj=TESTING_CONFIG):
     from .api import setup_routing
     app = Flask(__name__)
     app.config.from_object(config_obj)
     setup_routing(FlaskExtensions.api)
     with app.app_context():
         FlaskExtensions.api.init_app(app)
         FlaskExtensions.db.init_app(app)
+        FlaskExtensions.db.create_all()
+    @app.route('/', methods=['GET','POST'])
```

Comment:
Ideally this would be somewhere else, like a module called 'views' or something.

Reply:
Armaan says it works fine there.

Reply:
That wasn't the point. I know it works fine there. It's also sloppy. I'll allow it so that we can get this done, but I wanted to point out that the organization needs improvement.